src/test/java/seedu/address/logic/parser/MarkTaskCommandParserTest.java

```diff
@@ -16,8 +16,29 @@
 
 public class MarkTaskCommandParserTest {
 
+    /*
+    Test case design used: Equivalence Partition
```

Comment:
Great job following the course's recommendation on [EPs](https://nus-cs2103-ay2324s1.github.io/website/se-book-adapted/chapters/testCaseDesign.html#equivalence-partitions)!